parasolr/django/indexing.py

```diff
@@ -47,8 +48,7 @@ def index_data(self):
 
 
 @requires_django
-class ModelIndexable(Indexable):
-
+class ModelIndexable(models.Model, Indexable):
```

Comment:
hadn't we tried this? Surprised it didn't work the first time

Reply:
we did, but we didn't adjust the subclasses method to check for `._meta` instead of `.Meta`